src/quo2/components/wallet/transaction_progress/view.cljs

```diff
@@ -0,0 +1,298 @@
+        (= network-state :finalising)) ["positive-state"
+                                        (colors/theme-colors colors/white
+                                                             colors/success-50
+                                                             colors/success-60)]
+    (= network-state :finalized)       ["diamond"
+                                        (colors/theme-colors colors/white
+                                                             colors/success-50
+                                                             colors/success-60)]
+    (= network-state :error)           ["negative-state"
+                                        (colors/theme-colors colors/danger
+                                                             colors/danger-50
+                                                             colors/danger-60)]))
+
+(defn render-title
+  [network-state title override-theme]
+  [rn/view
+   {:style style/title-item-container}
+   [rn/view
+    {:style style/inner-container}
+    [load-icon "placeholder" (colors/theme-colors colors/white colors/neutral-50 colors/neutral-60)]
+    [rn/view
+     {:style style/title-container}
+     [render-text title override-theme]]
+    (when (= network-state :error)
+      [button/button
+       {:size      24
+        :icon-left :i/refresh
+        :type      :primary} "Retry"])]])
+
+(defn render-tag
+  [context-icon]
+  [rn/view
+   {:style style/padding-row}
+   [context-tag/context-tag {:blur? [false]}
+    (resources/get-mock-image context-icon)
+    "Doodle #120"]])
+
+(defn get-network-text
+  [network-type]
+  (cond
+    (= network-type :arbitrum) "Arbitrum"
+    (= network-type :mainnet)  "Mainnet"
+    (= network-type :optimism) "Optimism"))
+
+(defn render-status-row
+  [override-theme network-state network-type]
+  [rn/view
+   {:style style/item-container}
+   [rn/view
+    {:style (assoc style/progress-container
+                   :border-color
+                   (colors/theme-colors colors/white colors/neutral-10 colors/neutral-80))}
+    (let [[status-icon color] (get-status-icon network-type network-state)]
+      [load-icon status-icon color])
+    [rn/view
+     {:style style/title-container}
+     [render-text (str (network-type-text network-type network-state) (get-network-text network-type))
+      override-theme
+      :typography
+      :typography/font-regular :weight :regular :size :paragraph-2]]
+    [rn/view
+     [render-text (steps-text network-type network-state) override-theme :typography
+      :typography/font-regular :weight :regular :size :paragraph-2 :style
+      {:color (colors/theme-colors colors/white colors/neutral-50 colors/neutral-60)}]]]])
+
+(defn transaction-progress
+  [{:keys [title
+           on-press
+           accessibility-label
+           network-type
+           network-state
+           start-interval-now
+           context-icon
+           override-theme]}]
+  (let [count (reagent/atom 0)]
```

Comment:
This is not a valid form-2 component. You need to return a new anonymous function closing over the local state. The Reagent docs have more information about form-2 components. Also, I believe the linter won't allow you to use `count`, since it shadows the core clojure function.

Reply:
Here `count` is added by mistake, will remove that